libtransmission/quark.cc

```diff
@@ -386,6 +386,7 @@ auto constexpr MyStatic = std::array<std::string_view, TR_N_KEYS>{
     "torrent-complete-notification-enabled"sv,
     "torrent-complete-sound-command"sv,
     "torrent-complete-sound-enabled"sv,
+    "torrent-complete-verify-enabled"sv,
```

Comment:
This quark should be in snake_case.

@Coeur Can you do a follow-up PR?

Reply:
Everything else around is written like that.
I see that you have some quark_snake_case and quark_snake_case2 branches. Feel free to address it in those branches?

Reply:
I don't understand why does existing quarks being kebab-case matter here. I'm sure you know the project has decided that all new quarks will be snake_case quite a while ago.

But ok.